### PR TITLE
mostly admin updates

### PR DESCRIPTION
**Pull request by fitsum**
- **package updates**
- **add linter**
- **remove old lint rc**
- **add entry**
- **new file**
- **lint**
- **comment out default values for options + lint**
- **add NOTES with notes about commenting out default values in config**
- **uncomment default values because they don't seem to interfere with options anymore 🤷**
- **remove irrelevant notes**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
```
Comment (jaller94) [inline .eslintrc.json:3]: Mapscii doesn't use any browser APIs.
```suggestion
        "browser": false,
```

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
+        "array-bracket-spacing": [2, "always"],
+        "computed-property-spacing": [2, "always"],
+        "object-curly-spacing": [2, "always"],
+        "template-curly-spacing": ["error", "always"],
+        "space-in-parens": ["error", "always"]
```
Comment (jaller94) [inline .eslintrc.json:19]: This deviates from the current code style.

File: src/config.js [hunk 3]
```diff
@@ -55,5 +55,5 @@ module.exports = {
 
   delimeter: '\n\r',
 
-  poiMarker: '◉',
-};
+  poiMarker: '◉'
+}
```
Comment (jaller94) [inline src/config.js:59]: Removing trailing commas and semicolons deviates from the current code style.
I'd prefer to configure the linter to match the project's existing style.

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        "node": true
+    },
+    "extends": "standard",
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
```
Comment (jaller94) [inline .eslintrc.json:14]: The linter only seems to run on `main.js` and `src/config.js`. Many files in `src/` are in violation with the new linter rules, but have not been touched.

File: NOTES.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# NOTES
+- TBD
```
Comment (jaller94) [inline NOTES.md:2]: While the Issue Description on Github lists `add NOTES with notes about commenting out default values in config`, the committed file holds no information.

File: TODO.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  - eg. send old geoy data to mapscii via SEE
+- RESEARCH: options for tileSource
+- RESEARCH: stories: animated map + espeak TTS script
+- RESEARCH: add narrative/realtime detail to footer
+
+## TODO UI
+- TBD
+
+## TODO SERVICES
+- FEATURE: use app-stack
```
Comment (jaller94) [inline TODO.md:21]: What are these entries? A personal work list?
Why add another todo list? `Readme.md` already has one.
Why keep todos in the project files? The list in `Readme.md` has not been updated in over 5 years.
Why not on GitHub to link Pull Requests, for discussions or for other people to pick them up?

In the repo they won't have easy-to-see metadata like an author or a timestamp.

**Review by jaller94 — CHANGES_REQUESTED** [review]
Thank you very much for your interest in MapsCII and for contributing to this project. :+1: 

The addition of the NOTES and TODO files confuse me. Most development ideas are kept in issues on GitHub.

The update of the ESLint config seems very much needed to keep up with new ESLint releases. However, it seems that the rules are not matching the project's current code style and have not been applied to most code files in the `src` folder.

I wonder what rastapasta thinks.

File: .gitignore [hunk 7]
```diff
@@ -1,3 +1,4 @@
+.?**.sw*
```
Comment (jaller94) [inline .gitignore:1]: What kind of files are being ignored here?

Comment (fitsum) [reply .gitignore:1]: it's a poor practice for ignoring the temporary files vim creates
looks like I can just set a tmp directory in vimrc. I'm interested if you have a better solution. Thanks

File: TODO.md (hunk 6 above)
Comment (fitsum) [reply TODO.md:21]: Agreed. Adding my TODO and README was not appropriate for a PR. Thanks

File: NOTES.md (hunk 5 above)
Comment (fitsum) [reply NOTES.md:2]: Fair. Thanks

File: .eslintrc.json (hunk 4 above)
Comment (fitsum) [reply .eslintrc.json:14]: Fair. Thanks

**Comment by fitsum** [comment]
The rejection is fair. 
In the end I don't think the PR was as appropriate as a fork and actually meant to remove it
Thanks for your time reviewing this.

**Pull request closed without merging**